recreate_pods.sh

```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
 
-# Delete all pods that are owned by this RC.
-#  - Get the labels that the RC is selecting based on
-#  - Delete all the pods with that set of labels.
-#  - The RC will then recreate the pods.
-#
+# Update the required env vars for the first pod in a Deployments.
+# This will kick off a rolling update.
 # Do this so that the secrets can be remounted.
 
-if [ -z "$RC_NAMES" ]; then
-    echo "WARNING: RC_NAMES not provided. Secret changes may not be reflected."
+if [ -z "$DEPLOYMENTS" ]; then
+    echo "WARNING: DEPLOYMENTS not provided. Secret changes may not be reflected."
     exit
 fi
 
-RC_NAMES=(${RC_NAMES})
+DEPLOYMENTS=(${DEPLOYMENTS})
+DATE=$(date)
```

Comment:
What format is this in?
Should we be concerned that it should be timezone-normalized (e.g. GMT?) and non-localized. I guess in truth you aren't concerned about comparing dates for recency, just a string that has to change, it doesn't really matter.

On my system:

```bash
$ date 
dimarts, 19 de abril de 2016, 09:34:27 CEST

$ date -ju "+%F %T%z"
2016-04-19 07:36:38+0000
```